src/locutus/model/terminology.py

```diff
@@ -289,22 +312,24 @@ def delete_mappings(self, editor, code=None):
             mapping = tmref.get().to_dict()
             time_of_delete = 0
             if mapping is not None:
-                # This is not super helpful, but at least we get some detail about which mappings were removed
-                mapping = ",".join([x["code"] for x in mapping["codes"]])
+                # Iterate over the codes in the mapping and toggle 'valid' to False
+                for coding in mapping["codes"]:
+                    if "valid" in coding:

```

Comment:
Why are you only setting valid if it already exists? Shouldn't this assignment always happen?